conf/SoftwareProviders.json

```diff
@@ -4175,6 +4175,21 @@
     "hearing": false,
     "motor": false
   },
+  {
+    "name": "SAP Document and Reporting Compliance",
+    "url": "https://www.sap.com/uk/products/financial-management/document-reporting-compliance.html",
+    "business": true,
+    "agent": false,
+    "accounting": true,
+    "spreadsheets": true,
```

Comment:
Please atatch the screen shots of all the scenarios in the Jira ticket please.  Supported and not supported as attachment. Or this will be validated in Acceptance criteria?

Reply:
![Screenshot from 2024-10-15 13-32-37](https://github.com/user-attachments/assets/444c1c03-4878-4266-91d2-128f7ec95658)